src/App.js

```diff
@@ -5,19 +5,20 @@ import { NewNote } from "./Components/NewNote/NewNote";
 import { MainPage } from "./Pages/MainPage/MainPage";
 import { LandingPage } from "./Pages/LandingPage/LandingPage";
 
-import { PinNote } from "./Components/PinNote/PinNote.jsx";
-
 import { AllNotes } from "./Components/AllNotes/AllNotes.jsx";
 
 import { RecycleBin } from "./Components/RecycleBin/RecycleBin.jsx";
+import MockmanEs from "mockman-js";
+import { Archive } from "./Components/Archive/Archive.jsx";
 
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<LandingPage />} />
+        <Route path="/mockman" element={<MockmanEs />} />
         <Route path="/main-page" element={<MainPage page={<NewNote />} />} />
-        <Route path="pin-note" element={<MainPage page={<PinNote />} />} />
+        <Route path="archive" element={<MainPage page={<Archive />} />} />
 
         <Route path="/all-notes" element={<MainPage page={<AllNotes />} />} />
```

Comment:
i like the use of page attribute inside the element attribute.

Reply:
Thank you for the suggestion. Will keep in mind while refactoring the code.